examples/template.yaml

```
@@ -19,7 +19,7 @@ commands:
       Runs hello command by default based on GLOBAL_VAR.
     template: common_params
     run:
-    - $ {{GLOBAL_VAR}} f{local_arg} --f{local_arg_2}
+    - $ echo "{{GLOBAL_VAR}} f{local_arg} --f{local_arg_2}"
```

yamllint (1.35.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)